tasks/main.yaml

```diff
@@ -13,6 +12,9 @@
     gpgkey: "{{ clamav_epel_yumrepo_gpgkey }}"
     params: "{{ clamav_epel_yumrepo_params }}"
   when: clamav_epel_install == true
+  register: result
+  until: result is succeeded
+  retries: "{{ clamav_network_max_retries }}"
```

Comment:
By default the retry is 0, which does not loop, i.e.

```yaml
- hosts: localhost
  tasks:
    - get_url:
        url: https://example.com/does-not-exist
        dest: /dev/null
      register: r
      retries: 0
      until: r is succeeded
```

```
PLAY [localhost] **************************************************************************************************************************************************

TASK [Gathering Facts] ********************************************************************************************************************************************
ok: [localhost]

TASK [get_url] ****************************************************************************************************************************************************
fatal: [localhost]: FAILED! => changed=false 
  attempts: 1
  dest: /dev/null
  elapsed: 0
  gid: 0
  group: root
  mode: '0666'
  msg: Request failed
  owner: root
  response: 'HTTP Error 404: Not Found'
  size: 0
  state: file
  status_code: 404
  uid: 0
  url: https://example.com/does-not-exist

PLAY RECAP ********************************************************************************************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
```

This allows admins to improve durability with flakey DNS resolvers/networks. I retry all network tasks 3 times before finally failing since you can't always predict how nameservers in /etc/resolv.conf behave.